core/sourcehook/test/AMBuilder

```diff
@@ -7,6 +7,8 @@ for arch in MMS.archs:
   binary.compiler.cxxincludes += [
     os.path.join(builder.sourcePath, 'core', 'sourcehook'),
   ]
+  if binary.compiler.version >= 'gcc-4.9':
+    binary.compiler.cxxflags += ['-fno-devirtualize']
```

Comment:
Could we pass this when compiling the SH test files only?

Reply:
We could probably detect travis/appveyor builds through env vars in the build script, not sure if that's the best route though.

Reply:
Agree, having it for tests only seems ideal.